Imagine/Cache/Resolver/WebPathResolver.php

```diff
@@ -143,25 +143,14 @@ protected function getFileUrl($path, $filter)
      */
     protected function getBaseUrl()
     {
-        $port = '';
-        if ('https' === $this->requestContext->getScheme() && 443 !== $this->requestContext->getHttpsPort()) {
-            $port = ":{$this->requestContext->getHttpsPort()}";
-        }
-
-        if ('http' === $this->requestContext->getScheme() && 80 !== $this->requestContext->getHttpPort()) {
-            $port = ":{$this->requestContext->getHttpPort()}";
-        }
-
         $baseUrl = $this->requestContext->getBaseUrl();
         if ('.php' === mb_substr($this->requestContext->getBaseUrl(), -4)) {
             $baseUrl = pathinfo($this->requestContext->getBaseurl(), PATHINFO_DIRNAME);
         }
         $baseUrl = rtrim($baseUrl, '/\\');
 
-        return sprintf('%s://%s%s%s',
-            $this->requestContext->getScheme(),
+        return sprintf('//%s%s',
             $this->requestContext->getHost(),
-            $port,
```

Comment:
this will break if a site runs on a non-default port. i think we do need to keep the port handling.